README.md

```diff
@@ -375,6 +375,7 @@ The format specifiers are substitution based and work as follows:
 - `{M}`: The minute (00-59) of the start date\time.
 - `{S}`: The second (00-59) of the start date\time.
 - `{duration}`: The programme duration + any start and end buffer (if set).
+- `${duration}`: Same as above with a different syntax
```

Comment:
Should read: “Same as `{duration}`.”

If you use phrases such as, same as above and the line is moved it loses context.